spec/rails_helper.rb

```diff
@@ -9,17 +9,10 @@
 if Rails.env.production?
   abort("The Rails environment is running in production mode!")
 end
-require "coveralls"
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.